### PR TITLE
Listing Popups

### PR DESCRIPTION
**Pull request by jrgarou**
This PR adds:
- New Listing Popup `components/listings/show/Popup/index.js`
- ListingSlider with same design with Popup
- CSS icon setup to the ` components/shared/Common/Buttons` and `components/shared/Common/Buttons/CloseButton`

# Previews 
**Desktop:**
image
image

image
image
image

**Mobile:**
image
image
image
image
image

### CODE REVIEW

File: components/listings/show/Body/ListingInfo/styles.js [hunk 1]
```diff
@@ -52,61 +52,23 @@ export const ButtonsContainer = styled(Row)`
...
-      vertical-align: -0.125em;
-      font-size: inherit;
-    }
   }
 `
 
-export const OpenMatterportButton = styled(Button)`
-  z-index: 1;
+export const OpenMatterportButtonWrapper = styled.div`
+  z-index: 5;
```
Comment (lucas-tulio) [inline components/listings/show/Body/ListingInfo/styles.js]: Does it make sense to use the z-index constants here?

File: components/listings/show/ListingSlider/index.js [hunk 2]
```diff
@@ -183,32 +176,57 @@ class ListingGallery extends Component {
           }
         },
         {
-          breakpoint: 600,
+          breakpoint: 760,
```
Comment (lucas-tulio) [inline components/listings/show/ListingSlider/index.js]: Maybe we can use the breakpoint values from `ui`.

File: components/listings/show/ListingSlider/styles.js [hunk 3]
```diff
@@ -8,37 +8,56 @@ import {mobileMedia} from 'constants/media'
...
+import Text from '@emcasa/ui-dom/components/Text'
 
 export const SPINNER_SIZE = 40
 export const LISTINGSLIDER_HEIGHT = 454
 export const LISTINGSLIDER_OFFSET = (desktopHeaderHeight + listingDetailsBarHeight)
-export const LISTINGSLIDER_NAVIGATION_HEIGHT = 70
+const ARROW_SIZE = 48
 
 export default styled.div`
-  z-index: ${({isFullScreen}) => isFullScreen ? '9999' : null};
```
Comment (lucas-tulio) [inline components/listings/show/ListingSlider/styles.js]: real-life-css

File: components/listings/show/ListingSlider/styles.js [hunk 4]
```diff
@@ -8,37 +8,56 @@ import {mobileMedia} from 'constants/media'
...
+      top: ${theme.space[1]}px;
+      right: 0;
+    }
+  }
+
+  .slick-list {
+    width: 100%;
+    margin: 0 auto;
+
+    @media screen and (min-width: 813px) {
```
Comment (lucas-tulio) [inline components/listings/show/ListingSlider/styles.js]: Do these media queries need these specific values (812/813)? If not, we could use `ui` breakpoints.

If we have to use these values, we could move them to constants with descriptive names.

File: components/listings/show/Popup/styles.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    position: relative;
+    margin: 0;
+  }
+`
+
+export const Content = styled.div`
+  flex: 1 1 100%;
+  width: 100%;
+
+  @media screen and (min-width: 813px) {
```
Comment (lucas-tulio) [inline components/listings/show/Popup/styles.js]: Use constants here? Or take values from `ui` breakpoints.

File: components/shared/Common/Buttons/CloseButton/style.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+import theme from '@emcasa/ui'
+import Button from 'components/shared/Common/Buttons'
+
+export default styled(Button)`
+  z-index: 5;
```
Comment (lucas-tulio) [inline components/shared/Common/Buttons/CloseButton/style.js]: Maybe use z-index constants.

File: components/listings/show/Body/ListingInfo/styles.js (hunk 1 above)
Comment (jrgarou) [reply components/listings/show/Body/ListingInfo/styles.js]: No here. In fact, it should be an `2`, just to be above the elements in same level.

File: components/listings/show/ListingSlider/styles.js (hunk 4 above)
Comment (jrgarou) [reply components/listings/show/ListingSlider/styles.js]: They're very specific for landscape/mobile. It should not be used in `ui` for now I guess.

**Pull request merged**